### PR TITLE
[21818] Improve topic view

### PR DESCRIPTION
**Pull request by Carlosespicur**
# Main changes
- DomainGraph shows more compact, intercalating topic tags
- Added intercalated colors for Topic tags, topic bars and connection arrows .
- Fixed tag size. Larger tag names are elided and a hover shows the complete name

This PR must be merged after:
- https://github.com/eProsima/Fast-DDS-monitor/pull/238

### CODE REVIEW

File: qml/GraphConnection.qml [hunk 1]
```diff
@@ -81,10 +82,18 @@ Item {
         anchors.left: parent.left; anchors.leftMargin: left_arrow_margin_
 
         IconSVG {
+            id: left_arrow_icon
             name: "left_arrow"
             color: "grey"
             size: arrow_size_
         }
```
Comment (depink5) [inline qml/GraphConnection.qml:89]: Delete ID, to maintain format with the 'right_arrow'
```suggestion
        IconSVG {
            name: "left_arrow"
            color: "grey"
            size: arrow_size_
        }
```

**Review by depink5 — DISMISSED** [review]
LGTM with green CI 🎉

**Review by depink5 — APPROVED** [review]

**Review by depink5 — APPROVED** [review]
LGTM

**Review by rsanchez15 — CHANGES_REQUESTED** [review]
The box for the topic name keep the maximum width restriction even if the graph is in the filter view. I'd remove this restriction in that case.

image

**Pull request merged**